### ISSUE TITLE
Link?

### ISSUE THREAD

**Issue by whippingdot** [elipie/Piecademy#1]
I wanna ask, where is the link for this? Can't find it.

**Comment by Marcosultymos** [comment]
> I wanna ask, where is the link for this? Can't find it.

I don't think it has a link yet, but you can view it by opening the index.html file in your browser.

**Comment by whippingdot** [comment]
How? Do I have to fork the repo?

**Comment by Marcosultymos** [comment]
> How? Do I have to fork the repo?

You can clone using git or git cli, you also have the option to download from github (Pc version), after doing one of these, just open the in.dex.html file in a browser.
